PREOPS-3388: Write a technote describing gateway for releasing first alerts to brokers (#2)
lsst/rtn-061: RTN-061.tex

Accordingly, the commencement of routine Alert Production is a key gateway for early community science \citedsp{RTN-011}.
The Rubin project aims to release high-quality alerts as soon as is feasible.
Once Alert Production begins, it is expected to continue without interruption through the ten-year duration of the LSST survey.

Because image differencing relies on coadded template images, multiple prior observations in the same area of the sky are necessary to generate alerts at that position.
Throughout commissioning and the first year of the LSST survey we plan to build up templates incrementally as the data quality and availability allow \citedsp{RTN-011}.
The volume of alerts produced each night will thus scale up gradually with template availability.
We must balance generating templates and alerts as early as possible (to provide lengthy lightcurves and early followup) with providing larger volumes of higher quality alerts later (by building deeper, better sampled templates with fewer artifacts).

Necessarily, there will be a night on which the very first LSST alerts are released. 

> As this is likely to occur during the fluid period of commissioning but begins a long-term operational campaign, this document aims to specify gateway criteria for beginning Alert Production.

Reviewer: ```suggestion
As this is likely to occur during the fluid period of commissioning and begins a long-term operational campaign, this document aims to specify gateway criteria for beginning Alert Production.
```

> Its goal to to aid Project planning and decisionmaking while clarifying expectations for community scientists and our alert broker partners.

Reviewer: ```suggestion
Its goal is to aid Project planning and decision making while clarifying expectations for community scientists and our alert broker partners.
```

% cites: LDM-612, early science paper, SFL milestone, ...

\section{Technical Prerequisites}

We begin by describing at a high level the major technical capabilities required for routine Alert Production.

> For operational efficiency, we do not intend to release significant amounts of ``alert'' data outside of the normal Alert Production process.

Reviewer: Do you mean "release significant numbers of alerts" or "release significant amounts of alert-related" data. I worry about setting any expectation of alert packets being released outside of normal AP  processes.  Suggest removing this. If we end up doing it, it will be a bonus.

\subsection{Science Pipelines}

Alert production begins with pipeline software which differences new science images against coadded templates and associates the resulting DIASources against static DIAObjects as well as Solar System Objects (SSObjects).

% incl SSP association

\subsection{Prompt Processing and the USDF}

> The Science Pipelines payloads should be executed by the automated real-time Prompt Processing framework at the USDF.

Reviewer: ```suggestion
The Science Pipelines payloads will be executed by the automated real-time Prompt Processing framework at the USDF.
```

Reviewer: Is there any reason we think they might not be executed by the automated real-time Prompt Processing framework at the USDF?

Author: I'm using "should" here in the sense that it is a prerequisite for starting routine AP--the statement is intended to be prescriptive.  Indeed, we're not planning to run AP outside of PP (aside potentially from rare catch-up or replacement processing).

Reviewer: OK, that makes sense

\subsection{APDB and PPDB}

Realtime association of DIASources to DIAObjects and SSObjects requires a high-performance database, the Alert Production Database (APDB).
Records from the APDB must be replicated to a user-facing Prompt Processing Database (PPDB) \citedsp{DMTN-268}.

\subsection{Alert Archive}

> All transmitted alerts should be archived at the USDF \citedsp{DMTN-183}.

Reviewer: should or will?

Author: As above, I think "should."
